docs/BesuProxyModel.md

```diff
@@ -0,0 +1,43 @@
+
+**Testing Avalon Proxy model with Hyperledger Besu**
+1.  Install ``truffle`` using the command
+    ```
+    npm install -g truffle
+    ```
+    This requires Requires ``NodeJS v8.9.4 or later`` preinstalled.
+
+2.  Set the environment variable ``TCF_HOME`` to the Avalon root directory
+
+3.  Start the Ethereum network with 2 nodes and 2 eth-signers. To bring up the network, do the following
+    ```
+    cd $TCF_HOME/docs/dev-environments/besu
+    docker-compose up -d
+    ```
+       This will start the Ethereum network locally.
+
+4.  Now you need to create a truffle project and deploy contracts onto the Ethereum network
+       ```
+        mkdir ./my_project
+        cd my_project
+        cp $TCF_HOME/sdk/avalon_sdk/ethereum/contracts/proxy-model/*.sol ./contracts/
+        cp $TCF_HOME/sdk/avalon_sdk/ethereum/truffle_artifacts/truffle-config.js ./
+        cp $TCF_HOME/sdk/avalon_sdk/ethereum/truffle_artifacts/2_deploy_contracts.js ./migrations/
```

Comment:
Change `truffle_artifacts/2_deploy_contracts.js` to `truffle_project/migrations/2_deploy_contracts.js`